### PR TITLE
[HYDRATOR-1246] fixing null values

### PR DESCRIPTION
**Pull request by russorat**
fixing https://issues.cask.co/browse/HYDRATOR-1246

### CODE REVIEW

File: src/main/java/co/cask/hydrator/plugin/DateTransform.java [hunk 1]
```diff
@@ -99,7 +99,8 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
       String sourceField = sourceFields.get(i);
       String targetField = targetFields.get(i);
 
-      if (input.getSchema().getField(sourceField) == null) {
+      if (input.getSchema().getField(sourceField) == null ||
```
Comment (abhinavbansal246) [inline src/main/java/co/cask/hydrator/plugin/DateTransform.java]: Shouldn't it fail when the fields present in source fields config is not present in the input schema. We should check it while configuring a pipeline, somewhere here
https://github.com/hydrator/date-transform/blob/a0a64f1878ed5b75fc3212e3b1457821f3f41371/src/main/java/co/cask/hydrator/plugin/DateTransform.java#L255

```
if (inputSchema.getField(field) == null) {
throw exception;
};
```

Comment (abhinavbansal246) [inline src/main/java/co/cask/hydrator/plugin/DateTransform.java:102]: Shouldn't it fail when the fields present in source fields config is not present in the input schema. We should check it while configuring a pipeline, somewhere here
https://github.com/hydrator/date-transform/blob/a0a64f1878ed5b75fc3212e3b1457821f3f41371/src/main/java/co/cask/hydrator/plugin/DateTransform.java#L255

```
if (inputSchema.getField(field) == null) {
throw exception;
};
```

Comment (russorat) [reply src/main/java/co/cask/hydrator/plugin/DateTransform.java:102]: why?

**Comment by sreevatsanraman** [comment]
LGTM

**Pull request merged**